src/AddressRegistry.Api.BackOffice.Abstractions/Requests/AddressChangePositionRequest.cs

```diff
@@ -13,7 +13,7 @@ namespace AddressRegistry.Api.BackOffice.Abstractions.Requests
     using Swashbuckle.AspNetCore.Filters;
 
     [DataContract(Name = "WijzigenAdresPositie", Namespace = "")]
-    public class AddressChangePositionRequest : AddressBackOfficeChangePositionRequest, IRequest<ETagResponse>
+    public class AddressChangePositionRequest : BackOfficeChangePositionRequest, IRequest<ETagResponse>
```

Comment:
Personally, I'd like to see ChangeAddressPositionBackOfficeRequest with BackOffice at the end and the subject (being Address) mentioned.